certified-connectors/Converter/apiProperties.json

```diff
@@ -0,0 +1,22 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Api-Key (free on \"https://power2apps.de/apikey\")",
+          "description": "\"Api-Key (free on \"https://power2apps.de/apikey\")\" f\u00fcr diese API",
+          "tooltip": "Stellen Sie \"Api-Key (free on \"https://power2apps.de/apikey\")\" bereit.",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#FFFFFF",
```

Comment:
@stsiek Please refer the template provided for apiProperties.json file and update it to the default iconBrandColor.

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json